geomet_data_registry/store/__init__.py

```diff
@@ -111,6 +112,35 @@ def set_key(ctx, key, config):
     click.echo('Done')
 
 
+@click.command('get')
+@click.pass_context
+@click.option('--key', '-k', help='key name to retrieve from store')
+def get_key(ctx, key):
+    """get key from store"""
+
+    if all([key is None]):
+        raise click.ClickException('Missing --key/-k')
+
+    provider_def = {
+        'type': STORE_TYPE,
+        'url': STORE_URL
+    }
+
+    st = load_plugin('store', provider_def)
+
+    try:
+        click.echo('Getting {} key from store ({})..'.format(key, st.url))
```

Comment:
`click.echo('Getting {} key from store ({}).'.format(key, st.url))`

Reply:
Fixed this and improved the echo in the set command as well :).